edb/repl/__init__.py

```diff
@@ -642,6 +646,7 @@ def _command_list_scalar_types(
                           f'{eql_quote.quote_literal(pattern)}.')
                 elif 'system' not in flags:
                     print(R'No user-defined sclar types found. Try \lTS.')
+                self._command_semicolon_hint(arg)
```

Comment:
This should be in the `if pattern` block

Reply:
OK